app/assets/stylesheets/application.css.scss

```diff
@@ -11,19 +12,27 @@ $bg-color: #F9F9F9;
 $dark-bg: #2A2C33;
 $active-dark: #1C1F23;
 $active-accent: #FF2849;
+$highlight: rgba(255, 56, 77, 0.34);
+$bun: #DC8847;
+$softgrey: #717171;
+$muteWhite: #f7f7f7;
 
 body {
-  background: white;
+  background: $muteWhite;
 }
 
 body h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
   font-family: monospace ;
   font-weight: 500;
   line-height: 2;
-  color: #050505;
+  color: $softgrey;
 }
 
-#logo {
+#notice {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line